core/src/main/resources/templates/envs.html

```diff
@@ -450,7 +450,7 @@ <h4 class="mb-3">Shortcuts</h4>
 
 			<!-- Row -->
 
```

Comment:
Do you want to add a column here for the rest api or an icon to do a quick copy of the rest api?
Or happy to have it display in the cluster only?

Reply:
I would like to gave 2 copy buttons,  one fir other id too.. may be in another small pr